### DISCUSSION TITLE
Question about mitochondrial genomes

### DISCUSSION THREAD

**Discussion by amaeda16** [kblin/ncbi-genome-download#188] [Q&A]
Hello!

Congratulations on developing this neat solution to work with the new FTP directory. It's a time and lifesaver!

I wanted to ask if there is a way to batch download only complete mitochondrial genomes? I am going to be developing some new primers for metabarcoding, so it would be handy to see if the script can be tweaked to do it. 

I already read the way to specify the download for other groups such as vertebrates or plants, so I think that is fine for the moment. 

Thanks in advance!

**Comment by kblin** [comment]
I'm afraid the NCBI doesn't make mitochondria available separately at https://ftp.ncbi.nih.gov/genomes/refseq/

That said, you cam download all mitochondria at https://ftp.ncbi.nlm.nih.gov/refseq/release/mitochondrion/ and might be able to find an file describing if they are complete or not.

**Comment by npsonis** [comment]
Hi, currently mitochondria are available in https://ftp.ncbi.nih.gov/genomes/refseq/.
Any chance to included it as an option?
The same for plasmid, plastid...
